src/com/example/ui.cljs

```diff
@@ -91,7 +93,26 @@
            {:header (comp/get-query Header)}
            {:quiz (comp/get-query Quiz)}]
    :initial-state {:root/user {} :header {} :quiz {}}}
-  (div :.container.mx-auto.text-gray-100
+  (div :.text-gray-100
        (ui-header (comp/computed (:header props) {:on-signout #(comp/transact! this [(mut/delete-root-user nil)]) :user user}))
-       (when (:quiz props)
-         (ui-quiz (:quiz props)))))
+       (when (not (:user/authenticated? user))
+         (div :.grid.md:grid-cols-2.grid-cols-1
+              (div :.md:grid-span-1.md:min-h-screen.w-full.py-8.bg-gray-600.rounded.mb-16
+                   (div :.flex.flex-col.place-content-center.h-full.mb-8
+                        (h1 :.md:mb-24.mb-12.self-center.w-96.text-3xl "Track how you feel so you know when to train.")
```

Comment:
I probably would have done a dynamic sizing here but shadow cljs complaints when I use `w-1/3` syntax. doesn't like the special character.. I didn't look into it.

Reply:
yep, gotta use `{:className "w-1/3"}` if you want to use those classes... `/` has a special meaning 

https://clojureverse.org/t/how-are-you-styling-your-clojurescript-apps/7946/7#limitations-3

Reply:
I had a feeling it did, good to know